Makes/AppDelegate.m

```diff
@@ -0,0 +1,63 @@
+//
+//  AppDelegate.m
+//  Makes
+//
+//  Created by Mohamed Ragab on 1/8/17.
+//  Copyright © 2017 Mohamed Ragab. All rights reserved.
+//
+
+#import "AppDelegate.h"
+#import "MakesViewController.h"
+
+
+@interface AppDelegate ()
+{
+    MakesViewController* VC;
+}
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    // Override point for customization after application launch.
+    self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
+    VC = [[MakesViewController alloc] init];
```

Comment:
You can make it here MakesViewController *makesViewController = [[MakesViewController alloc] init];
Please note that vc is not a clear name